packages/hdwallet-core/tsconfig.build.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.build.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "./dist",
+    "rootDir": "src",
+  },
+  "include": ["src/**/*", "src/**/*.json"],
+  "exclude": ["node_modules", "**/*.test.ts", "**/*.spec.ts"],
+  "references": [
+    { "path": "../caip/tsconfig.build.json" },
```

Comment:
Why update the tsconfig here? Also, I don't think these paths are going to resolve properly.